### DISCUSSION TITLE
Question: Does DDS-Router support data deduplication

### DISCUSSION THREAD

**Discussion by deeplearningrobotics** [eProsima/DDS-Router#222] [Q&A] [answered]
Lets assume I have on one machine one publisher and on the other machine 10 subscribers. Those two machines are connected over ethernet. Is the data sent over ethernet once or 10 times?

And if deduplication is not supported yet, is it on the roadmap?

**Comment by rsanchez15** [comment, marked as answer]
Hi @deeplearningrobotics,

Let's assume you have the following scenario: 
```
1 Publisher <-> DDS Router 1 <-Ethernet-> DDS Router 2 <-> 10 Subscribers
```

The `DDS Router 1` will only send the data once to `DDS Router 2`. Then the `DDS Router 2` will behave as the publisher, sending the data to each subscriber. This is because the publisher and `DDS Router 1` are agnostic of the 10 subscribers, i.e., it only sees the subscriber of `DDS Router 1`.

**Reply by deeplearningrobotics** [reply]
Thank you so much!
